--- Rsamtools/Rsamtools/DESCRIPTION~	2013-04-18 07:46:40.000000000 +0200
+++ Rsamtools/Rsamtools/DESCRIPTION	2013-05-10 14:30:55.269037518 +0200
@@ -14,7 +14,7 @@
 LazyLoad: yes
 Depends: methods, IRanges (>= 1.15.35), GenomicRanges (>= 1.11.38),
         Biostrings (>= 2.25.6)
-Imports: methods, utils, IRanges, GenomicRanges, Biostrings, zlibbioc,
+Imports: methods, utils, IRanges, GenomicRanges, Biostrings,
         bitops, BiocGenerics (>= 0.1.3)
 Suggests: ShortRead (>= 1.13.19), GenomicFeatures,
         TxDb.Dmelanogaster.UCSC.dm3.ensGene, KEGG.db, pasillaBamSubset,
--- Rsamtools/Rsamtools/NAMESPACE~	2013-04-04 00:15:29.000000000 +0200
+++ Rsamtools/Rsamtools/NAMESPACE	2013-05-10 14:31:12.692547009 +0200
@@ -1,7 +1,5 @@
 useDynLib(Rsamtools, .registration=TRUE)
 
-import(zlibbioc)
-
 importFrom(bitops, bitAnd)
 
 importFrom(utils, read.table)           # normalizePath --> base in R-2.13
